apps/dashboard/src/components/new-group-stepper/general-info-step.tsx

```diff
@@ -206,7 +207,8 @@ export default function GeneralInfoStep({
                             _fingerprintDuration === undefined) ||
                         (group.type === "off-chain" && !_groupDescription) ||
                         (group.type === "off-chain" &&
-                            _groupDescription.length < 10)
+                            _groupDescription.length < 10) ||
+                        _fingerprintDuration < 0
```

Comment:
Good catch!

We could add this:

```suggestion
                       (group.type === "off-chain" && _fingerprintDuration < 0)
```

What do you think about adding this:

```ts
isDisabled={
                        !group.type ||
                        !_groupName ||
                        (group.type === "off-chain" &&
                            (!_groupDescription ||
                                _groupDescription.length < 10 ||
                                _fingerprintDuration === undefined ||
                                _fingerprintDuration < 0))
                    }
```